wifiphisher/common/dependencies.py

```diff
@@ -0,0 +1,43 @@
+"""Handle dependency check.
+
+This module checks for all the required dependency needed for full
+function.
+"""
+
+from collections import namedtuple
+from subprocess import (check_call, CalledProcessError)
+from wifiphisher.common.constants import DN
+
+
+Result = namedtuple("Result", ["status", "missing"])
+
+
+def is_installed(application):
+    # type: (str) -> bool
+    """Check if application is installed.
+
+    Return True if application is installed and False otherwise.
+    """
+    try:
+        check_call(["which", application], stdout=DN, stderr=DN)
```

Comment:
It's more reliable to use Python's `distutils.spawn.find_executable`.

Reply:
Furthermore, I'm not sure if a `which roguehostapd` is a sufficient check to know that `roguehostapd` has been successfully installed. Trying to import it is probably the surefire way. 

@anakin1028 may give us a hint on this.

Reply:
>It's more reliable to use Python's distutils.spawn.find_executable.

I tried using `distutils.spawn.find_executable` but `distutils` is not part of the standard library it seems. Checkout [this](https://github.com/wifiphisher/wifiphisher/pull/960/commits/b6bfff08ac200ee9682f90401acc4214328dcbe6) log.

Reply:
If use which to find roguehostapd, currently we will find nothing since roguehostapd is designed to be used as a library so I don't copy it to the bin folder. The most correct way is to check whether the shared library is get built in dist-packages but I think currently we can just check import roguehostapd is workable or not.